components/infobox/wikis/ageofempires/infobox_league_custom.lua

```diff
@@ -0,0 +1,337 @@
+	_league = league
+	league.createWidgetInjector = CustomLeague.createWidgetInjector
+	league.addCustomCells = CustomLeague.addCustomCells
+	league.defineCustomPageVariables = CustomLeague.defineCustomPageVariables
+	league.addToLpdb = CustomLeague.addToLpdb
+	league.getWikiCategories = CustomLeague.getWikiCategories
+
+	return league:createInfobox(frame)
+end
+
+function CustomLeague:createWidgetInjector()
+	return CustomInjector()
+end
+
+function CustomInjector:addCustomCells(widgets)
+	local args = _league.args
+
+	table.insert(widgets, Cell{
+		name = 'Game',
+		content = {Page.makeInternalLink(GameLookup.getName({args.game})) .. (args.beta and ' Beta' or '')}
+	})
+
+	table.insert(widgets, Cell{
+		name = 'Version',
+		content = {CustomLeague:_makeVersionLink(args)}
+	})
+
+	table.insert(widgets, Cell{
+		name = 'Patch',
+		content = {CustomLeague:_makePatchLink(args)}
+	})
+
+	table.insert(widgets, Cell{
+		name = 'Voobly & WololoKingdoms',
+		content = {args.voobly}
+	})
+
+	table.insert(widgets, Cell{
+		name = 'Game mode',
+		content = CustomLeague:_getGameModes(args)
+	})
+
+	table.insert(widgets, Cell{
+		name = 'Number of teams',
+		content = {args.team_number}
+	})
+
+	table.insert(widgets, Cell{
+		name = 'Number of players',
+		content = {args.player_number}
+	})
+
+	return widgets
+end
+
+function CustomInjector:parse(id, widgets)
+	local args = _league.args
+
+	if id == 'customcontent' then
+		if not String.isEmpty(args.map1) then
+			local map1mode = ''
+			if not String.isEmpty(args.map1mode) then
+				map1mode = MapMode.get({args.map1mode})
+			end
+
+			local maps = {Page.makeInternalLink(args.map1, args.map1 .. map1mode)}
+			local index  = 2
+
+			while not String.isEmpty(args['map' .. index]) do
+				local mapmode = ''
+				if not String.isEmpty(args['map' .. index .. 'mode']) then
+					mapmode = MapMode.get({args['map' .. index .. 'mode']})
+				end
+				table.insert(maps, '&nbsp;• ' ..
+					tostring(CustomLeague:_createNoWrappingSpan(
+						Page.makeInternalLink(args['map' .. index], args['map' .. index] .. mapmode)
+					))
+				)
+				index = index + 1
+			end
+
+			table.insert(widgets, Title{name = 'Maps'})
+			table.insert(widgets, Center{content = maps})
+		end
+
+		if not String.isEmpty(args.team1) then
+			local teams = {Page.makeInternalLink(args.team1)}
+			local index  = 2
+
+			while not String.isEmpty(args['team' .. index]) do
+				table.insert(teams, '&nbsp;• ' ..
+					tostring(CustomLeague:_createNoWrappingSpan(
+						Page.makeInternalLink(args['team' .. index])
+					))
+				)
+				index = index + 1
+			end
+
+			table.insert(widgets, Title{name = 'Teams'})
+			table.insert(widgets, Center{content = teams})
+		end
+	elseif id == 'prizepool' then
+		return {
+			Cell{
+				name = 'Prize pool',
+				content = {CustomLeague:_createPrizepool(args)}
+			}
+		}
+	elseif id == 'liquipediatier' then
+		return {
+			Cell{
+				name = 'Tier',
+				content = {CustomLeague:_createTier(args)}
```

Comment:
```suggestion
				content = {_EARNINGS_DISPLAY}
```

Reply:
Are you sure that is needed? rocketleague/infobox_league_custom and counterstrike/infobox_league_custom both don't do it that way.

Reply:
well, we want to be able to store it so we either have to do it this way or we need to adjust some stuff in the commons module
probably best to discuss it tomorrow when vogan is available